src/Features/CSharp/Portable/CodeRefactorings/MoveType/MoveTypeCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CodeRefactorings.MoveType;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings.MoveType
+{
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = PredefinedCodeRefactoringProviderNames.MoveTypeToFile), Shared]
+    internal class MoveTypeCodeRefactoringProvider : AbstractMoveTypeCodeRefactoringProvider
```

Comment:
Why do you need these subclasses?  Why not just define the CodeRefactoringProvider in the common project and export that for C# and VB?